assets/sass/_hubspot.scss

```diff
@@ -93,5 +93,17 @@
         label {
             @apply font-normal;
         }
+        input, textarea {
+            @apply appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight text-xl;
+            &:focus {
+                @apply outline-none shadow-input-outline;
+            }
+        }
+        .hs-button {
+            @apply text-white btn-orange;
```

Comment:
Let's have this button stick with the blue and unaltered font size (as with the submit button below), which should be the defaults: 

![image](https://user-images.githubusercontent.com/274700/69264431-2b634e00-0b7c-11ea-8fe8-97ea8859562d.png)